components/core/src/streaming_archive/writer/Archive.hpp

```diff
@@ -62,8 +65,8 @@ namespace streaming_archive { namespace writer {
             }
         };
 
-        TimestampPattern old_ts_pattern;
-
+        TimestampPattern m_old_ts_pattern;
```

Comment:
Could we use a pointer instead of an object? In all usages of it, it seems to me like we're simply comparing it against an existing timestamp pattern in `TimestampPattern`. If so, I think we can get rid of `m_timestamp_set` and use `nullptr`?

Reply:
Fixed in 96772cb9046081641441fd03502338becd0b41e8